net-im/prosody/prosody-0.10.2-r1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic multilib systemd
```

Comment:
you do not need multilib

Reply:
well, i didn't add multilib just adopt it from the previous versions to force an installation to lib64… not sure, if it's a regression when we remove it.

Reply:
@xdch47 it's not, because all you need is provided by >=EAPI-6.

Reply:
hopefully fixed (including some more cleanups + EPREFIX changes)